oracle/src/events/processSignatureRequests/index.js

```diff
@@ -30,18 +32,21 @@ function processSignatureRequestsBuilder(config) {
     rootLogger.debug(`Processing ${signatureRequests.length} SignatureRequest events`)
     const callbacks = signatureRequests
       .map(signatureRequest => async () => {
-        const { recipient, value } = signatureRequest.returnValues
+        const { recipient, value, nonce } = signatureRequest.returnValues
 
         const logger = rootLogger.child({
           eventTransactionHash: signatureRequest.transactionHash
         })
 
-        logger.info({ sender: recipient, value }, `Processing signatureRequest ${signatureRequest.transactionHash}`)
+        logger.info(
+          { sender: recipient, value, nonce },
+          `Processing signatureRequest ${signatureRequest.transactionHash}`
+        )
 
         const message = createMessage({
           recipient,
           value,
-          transactionHash: signatureRequest.transactionHash,
+          transactionHash: nonce,
```

Comment:
isn't better to rename `transactionHash` into `nonce` as `transactionHash` doesn't exist anymore?

Reply:
Solved in [b688329](https://github.com/gnosischain/tokenbridge/pull/5/commits/b688329b9028bdd7d43ee47bddd045ebb0ae2d27)